tensorflow-core-ops/Setup.hs

```diff
@@ -28,6 +28,9 @@ import Distribution.Simple
     , simpleUserHooks
     , UserHooks(..)
     )
+#if MIN_VERSION_Cabal(3,0,0)
+import Distribution.Utils.Path (unsafeMakeSymbolicPath, SymbolicPath)
```

Comment:
probably need to remove the `SymbolicPath` import to avoid an "unused import" warning

Reply:
Good point, done.